### PR TITLE
Add project support to security-issues

### LINKED ISSUE #111: ✨ Vulnerability issue creator must add issues to configurable project
# Summary
The Vulnerability issue creator creates issues for vulnerabilities in dependencies. These issues are currently not added to a GitHub project which would be necessary for our workflow.

# Details
After creating a new issue, the Vulnerability issue creator must add it to a configurable GitHub project.

# Out-of-scope
* Configuring the project attributes is out of scope for now, we can do this manually in the beginning.

### PR DESCRIPTION
**Pull request by Nicoretti**
Fixes #111 

# ✔ Checklist(s)

* [x] Is the title of the Pull Request correct?
* [x] Is the title of the corresponding issue correct?
* [x] Have you updated the changelog?
* [x] Have you updated the templates?
* [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?
* [x] Are you mentioning the issue which this PullRequest fixes ("Fixes...")

# 🦺 Github Actions
* [x] Did you update the version pinning in the action(s)
  * security-issues (exasol-toolbox)
    -> Only for releases!!

Note: If any of the above is not relevant to your PR just check the box.

### CODE REVIEW

File: .github/actions/security-issues/action.yml [hunk 1]
```diff
@@ -53,7 +57,7 @@ runs:
         GH_TOKEN: ${{ inputs.github-token }}
       shell: bash
       run: |
-        tbx security cve create < issues.jsonl | tee created.txt
+        tbx security cve create --project ${{ inputs.project }} < issues.jsonl | tee created.txt
```
Comment (tkilias) [inline .github/actions/security-issues/action.yml:60]: can be project a string with spaces?

File: doc/github_actions/security_issues.rst [hunk 2]
```diff
@@ -88,6 +88,12 @@ The temporary GitHub token of the workflow needs to be passed into the action (:
 in order to enable the action to query and created GitHub issues.
 
 
+project
++++++++
+Github project the crated issue(s) shall be associated with (default = None).
```
Comment (tkilias) [inline doc/github_actions/security_issues.rst:93]:
```suggestion
Github project the created issue(s) shall be associated with (default = None).
```

File: exasol/toolbox/tools/security.py [hunk 3]
```diff
@@ -235,6 +237,9 @@ def create(
     input_file: typer.FileText = typer.Argument(
         default="-", mode="r", help="file of cve's in the jsonl format"
     ),
+    project: str = typer.Option(
+        default="", help="Project the crated ticket shall be associated with."
```
Comment (tkilias) [inline exasol/toolbox/tools/security.py:241]:
```suggestion
        default="", help="Project the created ticket shall be associated with."
```

**Review by tkilias — CHANGES_REQUESTED** [review]

File: .github/actions/security-issues/action.yml (hunk 1 above)
Comment (Nicoretti) [reply .github/actions/security-issues/action.yml:60]: good point

File: .github/actions/security-issues/action.yml [hunk 4]
```diff
@@ -15,6 +15,10 @@ inputs:
     description: 'Github Token'
     required: true
 
+  project:
+    description: 'Project the created tickets shall be associated with'
```
Comment (kaklakariada) [inline .github/actions/security-issues/action.yml:19]: Add an example. Is this the project ID or name?

File: doc/github_actions/security_issues.rst (hunk 2 above)
Comment (kaklakariada) [reply doc/github_actions/security_issues.rst]: Is this the project ID or project name?

**Review by tkilias — APPROVED** [review]

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: .github/actions/security-issues/action.yml (hunk 4 above)
Comment (Nicoretti) [reply .github/actions/security-issues/action.yml:19]: Added the information to the documentation, see below.

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**